agent/eni/pause/pause_windows.go

```diff
@@ -0,0 +1,36 @@
+// +build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Why is the copyright missing the year?

Reply:
agent package changed the copyright message to not include years.